packages/react/src/themes/teams/components/Popup/popupContentVariables.ts

```diff
@@ -1,15 +1,33 @@
 import { pxToRem } from '../../../../lib'
 
 export interface PopupContentVariables {
-  [key: string]: string | number
-
   borderColor: string
+  borderRadius: string
+  borderSize: string
   padding: string
+
+  contentColor: string
```

Comment:
- there are `contentColor` and `contentBackgroundColor` left in `popupVariables` as well and are not used in `popupStyles`
- `popupVariables`in teams-dark is setting `color` which is not used
- if we move variables from `popup` to `popupContent` it is a breaking change

Reply:
`contentColor` and `contentBackgroundColor` are now variables of `PopupContent`. Is the **content** in the variables' names still meaningful?